src/test/java/pages/HomePage.java

```diff
@@ -23,6 +23,14 @@ public class HomePage extends BasePage{
     @FindBy(css = "[name='name']")
     WebElement playlistNameField;
```

Comment:
I recommend:  make all the **locators and WebElements private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    @FindBy(css = "[data-testid='sidebar-create-playlist-btn']")
    private WebElement createPlaylistButton;

    @FindBy(css = "[data-testid='playlist-context-menu-create-simple']")
    private WebElement newPlaylistOption;

    @FindBy(css = "[name='name']")
    private WebElement playlistNameField;

Reply:
**I recommend: make the variables protected in` class BasePage` ,** because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
public class BasePage {
    protected WebDriver driver;
    protected WebDriverWait wait;
    protected  Actions actions;
```

**Base Page \ Base Class**  - Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The method `isLoggedIn()` must be stored in the `class HomePage` because the avatar is visible on the web site -home page

![image](https://user-images.githubusercontent.com/104761482/204565125-444ca5aa-c6ab-465f-a269-3e781c65df39.png)

![image](https://user-images.githubusercontent.com/104761482/204565233-4a27219f-d0ad-457e-934c-0b50c66c5ea6.png)

![image](https://user-images.githubusercontent.com/104761482/204565301-40ec99ab-e994-43d4-8e2e-96dc4b0ec2ff.png)